examples/hikari_tanjun_basic_queue/requirements.txt

```
@@ -0,0 +1,3 @@
+hikari==2.0.0.dev102
+hikari-tanjun==2.1.0a1
+uvloop>=0.16; sys_platform != 'nt'
```

This seems to have been copy-pasted around a lot and idk where it originates from but sys_platform is actually "win32" for windows, not "nt" (ref https://docs.python.org/3/library/sys.html#sys.platform).

("nt" is used in sys.thread_info though)

---

Snab is correct here, sorry for this. Should b:
```suggestion
uvloop>=0.16; sys_platform != 'win32'
```

cc @Carberra and anybody else that I have messed this up with

---

Good callout snab.